assignment3/Controller/controller.go

```diff
@@ -0,0 +1,86 @@
+package controller
+
+import (
+	"fmt"
+	"github.com/anupriya0703/go/assignment3/domain"
+	"github.com/anupriya0703/go/assignment3/mapstore"
+	"net/http"
+	"github.com/gorilla/mux"
+)
```

Comment:
Import the package in the standard way like: 
1 - Inbuild library package
2 - Outside project library package
3 - Project-related package

Ex:- 
```
import (
	"fmt"
	"net/http"

	"github.com/gorilla/mux"

	"github.com/anupriya0703/go/assignment3/domain"
	"github.com/anupriya0703/go/assignment3/mapstore"
)
```